safetrace-sdk/Services/Session/UserSession.swift

```diff
@@ -90,6 +90,8 @@ class UserSession: UserSessionProtocol {
     }
 
     func logout() {
+        SafeTrace.stopTracing()
```

Comment:
can we use `environment.tracer` here? this is not going to be testable

Reply:
I see, so it feels like we should also move the API call for opting out into `environment.tracer`, because I want to hit that API when opting out